docs/content/modules/products/serverless-module.md

```diff
@@ -0,0 +1,382 @@
+---
+description: 'Learn how to use the product module in a serverless setup by installing it in a Next.js application.'
+addHowToData: true
+---
+
+# Serverless Products Module
+
+In this document, you’ll learn how to use the Products module in a serverless setup.
+
+## Overview
+
+Medusa’s modules increase extensibility and customization capabilities. Instead of having to manage a fully-fledged ecommerce system, you can use these modules in serverless applications, such as a Next.js project.
+
+The module only needs to connect to a PostgreSQL database. You can also connect it to an existing Medusa database. Then, you can use the Product module to connect directly to the PostgreSQL database and retrieve or manipulate data.
+
+This guide explains how to use the Product Module in a Next.js application as an example.
+
+:::note
+
+You can find a full sample Next.js project using the Product module in [this GitHub repository](https://github.com/medusajs/products-module-store).
+
+:::
+
+### Benefits of Serverless Modules
+
+- Keep packages small enough to be deployed to serverless Infrastructure easily
+- Bring Medusa closer to Edge runtime compatibility.
+- Make it easier to integrate modules into an ecosystem of existing commerce services.
+- Make it easier to customize or extend core logic in Medusa.
+
+:::info
+
+The product module is currently a work-in-progress, and it provides only functionalities related to browsing products. In later versions, the product module would include more powerful ecommerce features.
```

Comment:
In the article, we're calling it a beta release. Should we change work-in-progress to beta release here as well?